redwood/src/lib.rs

```diff
@@ -217,6 +217,39 @@ mod tests {
         assert_eq!(format!("{}", cert.fingerprint()), fingerprint);
     }
 
+    #[test]
+    fn test_source_key_creation_expiry() {
+        let (public_key, _secret_key, _fingerprint) =
+            generate_source_key_pair(PASSPHRASE, "foo@example.org").unwrap();
+        let cert = Cert::from_str(&public_key).unwrap();
+        let standard_creation_time = SystemTime::UNIX_EPOCH
+            // Equivalent to 2013-05-14
+            .checked_add(Duration::from_secs(1368507600))
+            // unwrap: Safe because the value is fixed and we know it won't overflow
+            .unwrap();
+        assert_eq!(
+            cert.primary_key()
+                .self_signatures()
+                .nth(0)
+                .unwrap()
+                .signature_creation_time(),
+            Some(standard_creation_time)
+        );
+
+        // From the documentation:
+        // This function is called signature_validity_period and not signature_expiration_time
+        // [because] the time is actually relative to the signature's creation time, which
```

Comment:
nit: I assume `[because]` is in brackets because in the upstream docs its a link, but since we don't have the link, remove the brackets so it's slightly easier to read?

Reply:
Oh, it's because I elided the documentation and am paraphrasing, but maybe I'm just being pedantic and annoying